core/src/main/scala-2/com/github/sbt/jni/Process.scala

```diff
@@ -3,8 +3,8 @@ package com.github.sbt.jni
 object Process {
   def out(command: String): String =
     try {
-      scala.sys.process.Process("uname -sm").lineStream.head
+      scala.sys.process.Process("uname -sm").!!.linesIterator.next()
```

Comment:
Hm, in this case we don't need `{scala-2/scala-3}.com.github.sbt.jni.Process`. I believe this syntax works in Scala 3 as well.

Reply:
Btw just to mentiont that explicitly: we can completely remove `com.github.sbt.jni.Process`